README.md

```diff
@@ -212,27 +201,28 @@ specific label files (see section below). Please type:
 
 to see all the options.
 
-Note that to replicate results exactly as in the paper you may need to checkout
-the specific version of NuPIC (and associated nupic.core) that is noted in the
-[Scoreboard](https://github.com/numenta/NAB/wiki/NAB%20Scoreboard):
+##### Run HTM with NAB
+
+In 2019 much of NAB was ported to Python 3, however NuPIC itself was not.
+
+NuPIC dependent detectors are available in the [NAB-detectors]() repository.
+
+Please refer to that repository on how to run any of the following detectors:
+
+    numenta
+    numentaTM
+    htmjava
```

Comment:
I don't think `htmjava` needed to be moved. It should run from its python 3 launcher script.

Reply:
The 'htmjava_detector.py' script imports from nupic:

https://github.com/iandanforth/NAB-detectors/blob/master/nab/detectors/htmjava/htmjava_detector.py#L25

Should that be removed?

Reply:
Subutai and I are discussing this, so don't move this back yet.

Reply:
That is too bad, we have to move it then. Good call.